packages/react-meteor-data/useFind.ts

```diff
@@ -42,8 +42,16 @@ const useFindReducer = <T>(data: T[], action: useFindActions<T>): T[] => {
   }
 }
 
+// Check for valid Cursor or null.
+// On client, we should have a Mongo.Cursor (defined in
+// https://github.com/meteor/meteor/blob/devel/packages/minimongo/cursor.js and
+// https://github.com/meteor/meteor/blob/devel/packages/mongo/collection.js).
+// On server, however, we instead get a private Cursor type from
+// https://github.com/meteor/meteor/blob/devel/packages/mongo/mongo_driver.js
+// which has fields _mongo and _cursorDescription.
 const checkCursor = <T>(cursor: Mongo.Cursor<T> | undefined | null) => {
-  if (cursor !== null && cursor !== undefined && !(cursor instanceof Mongo.Cursor)) {
+  if (cursor != null && !(cursor instanceof Mongo.Cursor) &&
```

Comment:
Shouldn't be enough checking for Meteor.isServer?

Reply:
The intent here is to print a warning if the argument is not an actual Meteor/Mongo cursor. I don't see how checking `Meteor.isServer` would suffice for that.

Reply:
I addressed this. It's good to go.

Reply:
Not a big deal, but `undefined` and `null` are the only two values that are `== null`. Reverting is also fine though; probably clearer not to use `==`.

Reply:
Ah the vagaries of Javascript's type marshalling. It's probably a bit clearer with explicit checks, but that's good to know, and now that I do know, I don't have a strong opinion either way.